src/network/networkd-address.c

```diff
@@ -448,6 +448,127 @@ static int address_set_masquerade(Address *address, bool add) {
         return 0;
 }
 
+static int netlabel_op(bool add, const char *label, const Address *address) {
+        _cleanup_(sd_netlink_unrefp) sd_netlink *genl = NULL;
+        _cleanup_(sd_netlink_message_unrefp) sd_netlink_message *m = NULL;
+        int r;
+
+        assert(address);
+        assert(address->link);
+        assert(address->link->network);
```

Comment:
Maybe, it is better to add `assert(IN_SET(address->family, AF_INET, AF_INET6));`